### ISSUE TITLE
Avoid looped open boundaries

### ISSUE THREAD

**Issue by SorooshMani-NOAA** [noaa-ocs-modeling/OCSMesh#103]
If only one open boundary is defined and that's the only boundary in the mesh, make sure it's not looped, otherwise it results in:

```
0: ABORT:  Looped open bnd:           1 
```

error from SCHISM. @feiye-vims @cuill Is this physically the right solution?

**Comment by feiye-vims** [comment]
I think that's just a requirement of SCHISM. I haven't dealt with such a case myself, so I'm not sure how you deal with the gap (between the last node and the first node) in the open boundary if you don't make a loop.

If the mesh only has one **land** boundary, I just arbitrarily break it into two pieces.

**Comment by SorooshMani-NOAA** [comment]
Thanks @feiye-vims, that makes sense. @josephzhang8 do you suggest that we split a single looped **open** boundary like what @feiye-vims is suggesting for **land**?

**Comment by josephzhang8** [comment]
Correct. Having only 1 segment on open/exterior land boundary is not allowed in SCHISM. This is allowed for island boundary, but there you do not put last node as 1st (and the code will understand it's a closed island segment).

**Comment by SorooshMani-NOAA** [comment]
@josephzhang8 Thanks, I'll account for this in future iterations of auto boundary generation in OCSMesh.
